parser/c2_tokenizer.c2

```diff
@@ -781,7 +775,7 @@ fn void Tokenizer.lex_internal(Tokenizer* t, Token* result) {
         case CR:
             t.cur++;
             if (*t.cur != '\n') {
-                t.error(result, "unexpected char 0x%02X", *t.cur);
+                t.error(result, "unexpected char 0x%02X after CR", *t.cur & 0xFF);
```

Comment:
Why AND with 0xFF, it can never be larger anyway since it's a char, maybe this needs to be 8F?